rocky/rocky/views/ooi_add.py

```diff
@@ -70,10 +70,17 @@ def get_ooi_class(self) -> type[OOI]:
         except KeyError:
             raise Http404("OOI not found")
 
+    def get_form_kwargs(self):
+        kwargs = super().get_form_kwargs()
+        kwargs["user_id"] = self.request.user.id
+
+        return kwargs
+
     def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
 
         context["type"] = self.ooi_class.get_ooi_type()
+        context["user_id"] = str(self.request.user)
```

Comment:
You're assigning the *user*, not the *user id*. The explicit `str` call is also not needed, the template renderer will always implicitly cast it to strings

```suggestion
        context["user"] = self.request.user
```